Sources/Tasks/ExportProfileData.php

```diff
@@ -1883,7 +1883,7 @@ public static function add_dtd(
 	 * Adjusts some parse_bbc() parameters for the special case of HTML and
 	 * XML_XSLT exports.
 	 */
-	public static function pre_parsebbc_html(string &$message, array &$smileys, string &$cache_id, array &$parse_tags, array &$cache_key_extras): void
+	public static function pre_parsebbc_html(string &$message, bool &$smileys, string &$cache_id, array &$parse_tags, array &$cache_key_extras): void
```

Comment:
In the `pre_parsebbc_xml` method that located below, you also need to use the `bool` type for `&$smileys`. Or not?

Reply:
Correct.  I will send it here, but I believe I have this fixed in my strict types PR.  Lots of issues found with ExportProfileData and other tasks.